samples/sampler/colors-toolbar/CustomStoryDecorator.module.less

```diff
@@ -0,0 +1,51 @@
+// CustomStoryDecorator styles
+
+@import '~@enact/sandstone/styles/mixins.less';
+
+.colorPickersBlock {
+  	width: 30%;
+	height: 65%;
+	position: absolute;
+	right: 51px;
+	bottom: 51px;
+	background: #2e3239;
+	border-radius: 30px;
+	padding: 12px;
+}
+
+.colorPicker {
+	color: #e6e6e6;
```

Comment:
Is there any idea a constant value can be gathered in a single file for CSS variables also? I don't have a clear idea yet. :(

Reply:
I created some local CSS variables in this file, so it's easier to maintain with future CSS updates